Test/UnitTest1.cs

```diff
@@ -0,0 +1,73 @@
+using Training;
+
+namespace MSTestProject
+{
+    [TestClass]
+    public class UnitTest1
+    {
+        MyList<int> myList = new ();
+
+        [TestMethod]
+        public void AddTest () {
+            myList.Add (1);

```

Comment:
Not formatted properly. The private fields of the class should be at the bottom of the class.